src/main/java/gregtech/client/shader/postprocessing/BloomType.java

```diff
@@ -0,0 +1,41 @@
+package gregtech.client.shader.postprocessing;
+
+import javax.annotation.Nonnull;
+
+public enum BloomType {
+    /**
+     * Simple Gaussian Blur
+     */
+    GAUSSIAN,
+    /**
+     * Unity Bloom
+     */
+    UNITY,
+    /**
+     * Unreal Bloom
+     */
+    UNREAL,
+    /**
+     * No bloom at all :O
+     */
+    DISABLED;
+
+    public int getValue() {
+        return switch (this) {
+            case GAUSSIAN -> 0;
+            case UNITY -> 1;
+            case UNREAL -> 2;
+            case DISABLED -> -1;
+        };
+    }
+
+    @Nonnull
+    public static BloomType fromValue(int value) {
```

Comment:
Instead of these two methods, I think we should use the enum in the config directly. Forge does natively support this and does also display a list of valid enum constants to choose in the config file.